src/manifolds/GraphManifold.jl

```diff
@@ -46,84 +46,83 @@ end
 const VertexGraphManifold = GraphManifold{<:AbstractGraph,<:Manifold,VertexManifold}
 
-@doc doc"""
-    check_manifold_point(M::GraphManifold, x)
+@doc raw"""
+    check_manifold_point(M::GraphManifold, p)
 
-Check whether `x` is a valid point on the [`GraphManifold`](@ref), i.e. its
-length equals the number of vertices (for [`VertexManifold`](@ref)s) or
-the number of edges (for [`EdgeManifold`](@ref)s) and that each element of `x`
+Check whether `p` is a valid point on the [`GraphManifold`](@ref), i.e. its length equals the number of vertices
+(for [`VertexManifold`](@ref)s) or the number of edges (for [`EdgeManifold`](@ref)s) and that each element of `p`
 passes the [`check_manifold_point`](@ref) test for the base manifold `M.manifold`.
 """
 check_manifold_point(::GraphManifold, ::Any...)
-function check_manifold_point(M::VertexGraphManifold, x; kwargs...)
-    if size(x) != (nv(M.graph),)
+function check_manifold_point(M::VertexGraphManifold, p; kwargs...)
+    if size(p) != (nv(M.graph),)
         return DomainError(
-            length(x),
-            "The number of points in `x` ($(length(x))) does not match the number of nodes in the graph ($(nv(M.graph))).",
+            length(p),
+            "The number of points in `x` ($(length(p))) does not match the number of nodes in the graph ($(nv(M.graph))).",
         )
     end
     PM = PowerManifold(M.manifold, NestedPowerRepresentation(), nv(M.graph))
-    return check_manifold_point(PM, x; kwargs...)
+    return check_manifold_point(PM, p; kwargs...)
 end
-function check_manifold_point(M::EdgeGraphManifold, x; kwargs...)
-    if size(x) != (ne(M.graph),)
+function check_manifold_point(M::EdgeGraphManifold, p; kwargs...)
+    if size(p) != (ne(M.graph),)
         return DomainError(
-            length(x),
-            "The number of points in `x` ($(size(x))) does not match the number of edges in the graph ($(ne(M.graph))).",
+            length(p),
+            "The number of points in `x` ($(size(p))) does not match the number of edges in the graph ($(ne(M.graph))).",
         )
     end
     PM = PowerManifold(M.manifold, NestedPowerRepresentation(), ne(M.graph))
-    return check_manifold_point(PM, x; kwargs...)
+    return check_manifold_point(PM, p; kwargs...)
 end
 
-@doc doc"""
-    check_tangent_vector(M::GraphManifold, x, v)
+@doc raw"""
+    check_tangent_vector(M::GraphManifold, p, X)
 
-Check whether `x` is a valid point on the [`GraphManifold`](@ref), and
-`v` it from its tangent space, i.e. its
+Check whether `p` is a valid point on the [`GraphManifold`](@ref), and
+`X` it from its tangent space, i.e. its
 length equals the number of vertices (for [`VertexManifold`](@ref)s) or
-the number of edges (for [`EdgeManifold`](@ref)s) and that each element of `v`
-together with its corresponding einty of `x` passes the
+the number of edges (for [`EdgeManifold`](@ref)s) and that each element of `X`
+together with its corresponding einty of `p` passes the
```

Comment:
einty?

Reply:
should've been `entry` I don't know how one can mistype it like that.